lib/index.js

```diff
@@ -62,12 +74,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       store: ['validate', (results, callback) => {
         // generate new random ID for inbox
         const inbox = results.validate;
-        inbox.id = routes.inboxes + '/' + uuidV4();
-        const { owner: owner=req.user.identity.id } = inbox;
```

Comment:
Unfortunately, eslint makes no distinction between an object literal and destructuring assignments.  So for my part,  I would like to use a style supported by the linting tool.